runtime/src/accounts_cache.rs

```diff
@@ -289,26 +289,21 @@ impl AccountsCache {
         removed_slots
     }
 
-    pub fn find_older_frozen_slots(&self, num_to_retain: usize) -> Vec<Slot> {
-        if self.cache.len() > num_to_retain {
-            let mut slots: Vec<_> = self
-                .cache
-                .iter()
-                .filter_map(|item| {
-                    let (slot, slot_cache) = item.pair();
-                    if slot_cache.is_frozen() {
-                        Some(*slot)
-                    } else {
-                        None
-                    }
-                })
-                .collect();
-            slots.sort_unstable();
-            slots.truncate(slots.len().saturating_sub(num_to_retain));
-            slots
-        } else {
-            vec![]
-        }
+    pub fn find_older_frozen_slots(&self) -> Vec<Slot> {
```

Comment:
nit: this just returns all frozen slots now, so would rename to `cached_frozen_slots`